src/compile/selection/interval.ts

```diff
@@ -29,11 +28,12 @@ const interval: SelectionCompiler<'interval'> = {
 
     if (selCmpt.translate && !hasScales) {
       const filterExpr = `!event.item || event.item.mark.name !== ${stringValue(name + BRUSH)}`;
-      events(selCmpt, (_: OnEvent[], evt: EventStream) => {
-        const filters = evt.between[0].filter || (evt.between[0].filter = []);
+      events(selCmpt, (on: OnEvent[], evt: Stream) => {
+        const filters = array(evt.between[0].filter || (evt.between[0].filter = []));
         if (filters.indexOf(filterExpr) < 0) {
           filters.push(filterExpr);
         }
+        return on;
```

Comment:
I had to add this to make the typings correct. Does JavaScript magically return the first argument here?